backend/.babelrc

```diff
@@ -1,5 +1,7 @@
 {
   "presets": [
     "@babel/preset-env"
-  ]
+  ],
+  "sourceMaps": "inline",
+  "retainLines": true
 }
```

Comment:
The lines above ensure we link the compiled code back to our source code. If we weren't using babel and just vanilla js this would not be needed